http-plugins/README.rst

```diff
@@ -5,31 +5,30 @@ URL Plugins
 Introduction
 ============
 
-A Hydrator Real Time Plugin that allows data to be fetched from a given URL at a specified interval.
+A Hydrator real-time plugin that allows data to be fetched from a given URL at a specified interval.
```

Comment:
I may actually just remove this readme.  There is one at the top level with almost all of this information (only non-overlap is the intro)